community/modules/scripts/ramble-setup/templates/ramble_setup.yml.tftpl

```diff
@@ -127,9 +133,24 @@
     when: lock_out.rc == 0
 
   - name: Wait for lock
-    ansible.builtin.wait_for:
-      path: "{{ lock_dir }}/done"
-      state: present
-      timeout: 600
-      sleep: 10
-    when: lock_out.rc != 0
+    block: 
+    - name: Wait for lock
+      ansible.builtin.wait_for:
+        path: "{{ lock_dir }}/done"
+        state: present
+        timeout: 600
+        sleep: 10
+      when: lock_out.rc != 0
+
+    rescue:
+    - name: Timed out on lock, get install directory contents
+      ansible.builtin.command: "ls -ltra {{ lock_dir }}"
+      register: lock_dir_contents
+
+    - name: Print install directory contents with host that failed to install spack
```

Comment:
I might name this instead `Print lock directory contents, it should contain name of host that is holding lock`

Reply:
Done